conda_mirror/conda_mirror.py

```diff
@@ -79,6 +84,7 @@ def _match(all_packages, key_glob_dict):
     matched : dict
         Iterable of package metadata dicts which match the `target_packages`
         (key, glob_value) tuples
+
```

Comment:
I think, I'm relatively used to following [Numpy's example](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) and probably followed `pep257` complaining at some point.

But I must admit that I did not check if this integrates with the existing docstring styles.